### PR TITLE
315: Bulk Edits

### LINKED ISSUE #315: Bulk Edits for Drop-downs
**Describe the task**
Bulk edit for any drop-downs. 

**Acceptance Criteria**
- [x] Assume you receive an array of activity IDs, and then do the right thing with them from there.
- [x] Advance filtering to create data subsets for which selected variable can be globally edited.
- [x] Creation of Bulk-edit button
- [x] Edit ability only applicable for drop-downs within set of records.
- [x] Restrict to same type of activity (1:1)

**Additional Context**
Look at Material UI tables when actioning this ticket

### PR DESCRIPTION
**Pull request by sdevalapurkar**
# Overview

This PR includes the following proposed change(s):

- Ability to bulk edit records of the same type
- Assumption is that a list of activity ids will be passed in as props (temporary hard coded but will be updated once multi select functionality is in place)
- Only specific dropdown fields are editable based on discussions with @mikeshasko 
- Temporarily only works with plant observation (terrestrial) and plant treatments - monitorings, animal records, and plant aquatic are to be added later once those forms are updated and finalized
- Terrestrial plant observation was misspelt so they wont work to be bulk edited for existing old activities. Newly created and synced ones will work though
- #315 

## Type of change

- [x] New feature (non-breaking change which adds functionality)

## How Has This Been Tested?

- Locally

## Screenshots

sa

### CODE REVIEW

File: app/src/components/activities-list/ActivitiesList.tsx [hunk 1]
```diff
@@ -354,7 +354,7 @@ const ActivitiesList: React.FC = (props) => {
                         addNewActivityToDB(
                           databaseContext,
                           ActivityType.Observation,
-                          ActivitySubtype.Observation_PlantTerrestial
```
Comment (sdevalapurkar) [inline app/src/components/activities-list/ActivitiesList.tsx:357]: this was spelt incorrectly and was causing issues for me for a while until I noticed the missing letter 😆

**Comment by dogcomplex** [comment]
I'm assuming you're still updating with lints etc here, but I took a look through the code and ran the branch - looks good so far!

**Comment by sdevalapurkar** [comment]
> I'm assuming you're still updating with lints etc here, but I took a look through the code and ran the branch - looks good so far!

No this is ready for review - no linting failures

**Review by micheal-w-wells — DISMISSED** [review]

**Review by dogcomplex — DISMISSED** [review]
Ah yes, the lints were on unchanged files - in that case, approved here too 👍

**Review by NickPhura — APPROVED** [review]

**Pull request merged**